### ISSUE TITLE
Cannot add new etcd node to k8s 1.17 cluster after an existing etcd node is powered down and deleted.

### ISSUE THREAD

**Issue by aaronRancher** [rancher/rancher#24641]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug
This bug is not seen in `k8s v1.16.3-rancher1-1`

**Steps to reproduce (least amount of steps as possible):**
Create a custom EC2 cluster with 3 etcd nodes, 1 control, and 2 worker nodes with `k8s v1.17.0-rancher1-1 (experimental)`
Stop  one of the etcd the node from AWS console. 
Delete this etcd node from Rancher UI.
Provision a new etcd node.

**Result:**
The new etcd node is stuck registering. The cluster is stuck updating.
![Screen Shot 2019-12-17 at 1 10 48 PM](https://user-images.githubusercontent.com/57409052/71031146-0e189580-20d0-11ea-8c67-c816450f102a.png)
```
2019/12/17 18:36:12 [ERROR] ClusterController c-vvh84 [cluster-provisioner-controller] failed with : Failed to reconcile etcd plane: Failed to create etcd client for host [172.31.15.238]: unexpected status code 404
```
A 404 is thrown on a different etcd node.

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): `v2.3-head` `version 78ed261c6`
- Installation option (single install/HA): Single

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Custom
- Kubernetes version (use `kubectl version`): `k8s v1.17.0-rancher1-1 (experimental)`

- Docker version (use `docker version`): 19.03

**Comment by superseb** [comment]
We use the v2 API to query etcd members and this is disabled by default in 3.4: https://github.com/etcd-io/etcd/pull/10935

Workaround until there is a fix is to use the following in the cluster config:

```
services:
  etcd:
    extra_args:
      enable-v2: "true"
```

Or for Rancher cluster API

```
"extraArgs":{"enable-v2":true}
```

**Comment by aaronRancher** [comment]
The workaround fixes this issue in `rancher\rancher:v2.3-head` `version 5e20b72cf`

**Comment by aaronRancher** [comment]
The bug fix is verified with Rancher server `rancher\rancher:master-head` `version 7578b3c76` and with `rancher\rancher:v2.3-head` `version 7ff247737`
Steps:
Create a custom EC2 cluster with 3 etcd nodes, 1 control, and 2 worker nodes with `k8s v1.17.0-rancher1-1 (experimental)`
Stop one of the etcd the node from AWS console.
Delete this etcd node from Rancher UI.
Provision a new etcd node.
